bbolt/bbolt.go

```diff
@@ -56,7 +61,26 @@ func createBBoltStore(filePath string, options *bbolt.Options, cfg stoabs.Config
 	if err != nil {
 		return nil, err
 	}
+
+	// log warning if file opening hangs
+	done := make(chan bool, 1)
+	if cfg.Log != nil {
```

Comment:
now it won't work when the logger isn't set, while later we use the standard logger. Might want to shuffle some things to get it still logged, or introduce a function for retrieving the logger so you can use it here as well

Reply:
Added a default logger to store that is now used when no logger is provided